platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/results/AggregateTestEventReporter.java

```diff
@@ -85,8 +87,16 @@ public void report(File reportDir, ProblemConsumer validationFailures) {
             if (numFailedResults.get() > 1) {
                 emitReport(reportIndexFile);
             }
+        } else {
+            // Delete any stale report
+            if (reportDir.exists()) {
```

Comment:
Isn't reportDir here buildDir or something like it?

Reply:
This should probably be reportLocation

Reply:
Ah should have been `reportLocation`